### ISSUE TITLE
Example files

### ISSUE THREAD

**Issue by NathanTBurgess** [neurosimata/seizy_ml#1]
Hello! I was wondering if you could provide example data. 

Based on what I'm reading in the training tutorial all training data should be concatenated into one .h5 file along with an label file. What is the format of this label file? will each time point be labeled as seizure or non-seizure? 

The workflow, as I am currently thinking of it.

1. convert a set of edf files to on h5 file

2. create an h5 file that matches the length of the concatenated h5 file made from edf files which identifies (at each time point) whether a seizure is occurring or not.

3. follow the training tutorial (the preprocessing seems to be built into the train function)

4. follow the main tutorial

**Comment by pantelisantonoudiou** [comment]
Hi thank you for you questions and sorry for the late reply! I did not receive a notification for this issue, but I have changed the settings now so I will be receiving new notifications.

Did you manage to get the workflow working? All the steps you highlighted are correct. One more thing that I found makes a big difference is normalizing the extracted features per file before concatenating. 

In the next couple of weeks or so I will be 1) updating seizyML to require each h5 recording file separately for training (plus adding more a few more app features) and 2) I will create a simple repo for converting edf files to h5.

**Comment by NathanTBurgess** [comment]
Wait, so after your program generates the features from the h5 file, I should normalize those features? Or write code to do that? Doesn't the feature generation happen in the train function?

**Comment by pantelisantonoudiou** [comment]
Hi Nathan, I just updated SeizyML. Updated documentation coming in the next few days.

To briefly answer your question, the software normalizes the features, so you don't need to do any normalization. You only have to convert your EEG files to _h5_ (for default: downsample to 100Hz) and create corresponding _csv_ files with ground truth labels (see below). As mentioned before, I have now created an additional toolbox https://github.com/neurosimata/seizy_convert that handles conversion of _edf_ files to the appropriate _h5_ format. But let me know if you need help with data conversion. Happy to help.

**To use SeizymL:**
- Train the model using 24-hour files containing seizures. I have tested it with **four**  files and the model captured all seizures in my test dataset.
- As before, use the trained model to generate predictions as explained in the tutorial [doc](https://github.com/neurosimata/seizy_ml/blob/main/docs/tutorial.md). 

**To train the model you will need:**
1) The converted h5 files (one per recording). These files are in the following format (Nsegments, 500 (win*fs), Nchannels). Nsegments = length of recording in samples/500. As mentioned above seizy_convert will do that for you if you have EDF files. 

2) A csv file with one column and length Nsegments containing the training labels (1 for seizure, 0 for non seizure) for each segment.

3) The h5 and csv files corresponding to the same recordings need to have a matching name. Here is an example of the train folder contents.
![example_files](https://github.com/user-attachments/assets/8f145fb8-88e1-478d-9af7-9bf7a9783bd0)

Let me know if you need additional questions.
